fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -470,33 +488,11 @@ def _setup_backward_hooks(self) -> None:
             p_tmp = param.expand_as(param)
             assert p_tmp.grad_fn is not None
             grad_acc = p_tmp.grad_fn.next_functions[0][0]
-            dst_rank = OSS.get_global_rank(self.process_group, self._trainable_param_to_rank[param])
+            dst_rank = self._trainable_param_to_rank[param]
```

Comment:
I thought this was a recent change to support multiple groups?

Reply:
yes, it's kept, but the buckets were not supported (this was also used to index the buckets)